script/test

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
```

Comment:
No longer needed - does the same thing as `make test-short` or `make test` that has more validation.